### PR TITLE
Add watches for CloudStack resources to controller

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by cxbrowne1207 — APPROVED** [review]

File: config/manifest/eksa-components.yaml [hunk 1]
```diff
@@ -3806,6 +3806,75 @@ spec:
                   name:
                     type: string
                 type: object
+              packages:
```
Comment (cxbrowne1207) [inline config/manifest/eksa-components.yaml:3809]: Looks good. qq: Why do we need this?

Comment (cxbrowne1207) [inline config/manifest/eksa-components.yaml:3809]: qq: Why do we need this?

**Review by cxbrowne1207 — COMMENTED** [review]
Looks good. Left a quick question

File: config/manifest/eksa-components.yaml (hunk 1 above)
Comment (taneyland) [reply config/manifest/eksa-components.yaml:3809]: This was added when I ran `make generate-manifests`, so it must've been added recently and we just never generated the new manifest then

**Pull request merged**